### DISCUSSION TITLE
Save/restore location window locations

### DISCUSSION THREAD

**Discussion by Stvad** [rxhanson/Rectangle#436] [Ideas]
It'd be great if one create snapshot of current state of windows (for everything or select subset of apps). And then restore those snapshots on-demand.

A bit more context - thinking on how one may do something like https://twitter.com/VladyslavSitalo/status/1380657822027837441?s=20

**Comment by SubtleCo** [comment]
Came here to say this! I finally found my favorite layout involving thirds and sixths. Would love a memory function.

**Comment by WolfgangFahl** [comment]
Wouldn't this be like the features of https://cordlessdog.com/stay/documentation/?

**Comment by nelsyeung** [comment]
Came here to say yes please! [This Stackoverflow](https://stackoverflow.com/questions/47480873/set-the-size-and-position-of-all-windows-on-the-screen-in-swift) makes it sound not too hard to implement too. Also, this is a feature of [Moom](https://manytricks.com/moom/).

**Comment by ppdc** [comment]
And one more voice for this feature!
